File: python/src/temporal_learn/process9/worker.py
import asyncio

from temporalio.client import Client
from temporalio.worker import Worker

from .models import TASK_QUEUE, TEMPORAL_URL
from .workflow import VerWorkflow


async def main():
    client = await Client.connect(TEMPORAL_URL)
    worker = Worker(
        client,
        task_queue=TASK_QUEUE,
        workflows=[VerWorkflow],
    )
    print(f"Worker started on {TASK_QUEUE}")
    await worker.run()


if __name__ == "__main__":
    asyncio.run(main())

File: typescript/src/process3/worker.ts
import { NativeConnection, Worker } from '@temporalio/worker';
import * as activities from './activities';
import { TASK_QUEUE, TEMPORAL_URL } from './models';

async function main() {
  const connection = await NativeConnection.connect({ address: TEMPORAL_URL });
  const worker = await Worker.create({
    connection,
    workflowsPath: require.resolve('./workflow'),
    activities,
    taskQueue: TASK_QUEUE,
  });

  console.log(`Worker started on ${TASK_QUEUE}`);
  await worker.run();
}

main().catch((err) => {
  console.error(err);
  process.exit(1);
});
